java/src/jmri/jmrix/ieee802154/xbee/ConnectionConfig.java

```diff
@@ -47,6 +47,7 @@ public String name() {
         return Bundle.getMessage("XBeeNetworkTitle");
     }
 
+    @SuppressWarnings("SameReturnValue")
     public boolean isOptList1Advanced() {
         return false;
     }
```

Comment:
What's the purpose of this method? If it always returns false. Or is it overridden by some class?

Reply:
It looks like it's an old method that needs to be cleaned up.

At one point it was part of the abstract class, but it doesn't appear to be called any longer.  There are still 13 instances in various ConnectionConfig files.  Some return true and some return false, but they all return a constant.

I'll submit a second PR to remove them all.

Reply:
Thanks!